app/src/main/res/values/strings.xml

```diff
@@ -35,6 +35,7 @@
     <string name="quest_leave_new_note_no">"No, just hide"</string>
     <string name="quest_leave_new_note_photo">Attach photo</string>
     <string name="quest_leave_new_note_photo_delete_title">Do you want to delete this picture?</string>
+    <string name="quest_leave_new_note_photos_are_useful">Adding a photo will make the note much more useful. Identifiers of location and direction like address plaques are very valuable.</string>
```

Comment:
Not sure what you mean with "identifiers". Also, it is quite long. What exactly is the main point you are trying to make with this note?

Reply:
That photo with address plaque is much more likely to be useful for a remote editor. Photo of a shop entrance is sometimes not good enough to actually edit without survey.

For example I would make edit based on https://westnordost.de/p/4162.jpg in https://www.openstreetmap.org/note/1836721 if it would have also an address plaque. Though maybe I should assume that it is in the same place as now nonexisting POI?

But I went trough several notes, and based on that I have seen: overall it is probably not worth making this message more complicated.